VueJS/index.js

```diff
@@ -1,13 +1,33 @@
 import List from './components/List.js'
 import ListItem from './components/ListItem.js'
 
+Vue.component('todo-list', List);
+
 new Vue({
   el: '#app',
   data: {
-    todos: [
+    idCounter: -1,
```

Comment:
Avoid state and use a computed property like here:
https://github.com/m-oehme/Systems-Development-and-Frameworks/pull/2#discussion_r339841486